docs/mfa/configuration.rst

```diff
@@ -7,6 +7,15 @@ Available settings:
   Specifies the adapter class to use, allowing you to alter certain
   default behaviour.
 
+``MFA_FORMS``
+  Used to override forms. Defaults to::
+
+    MFA_FORMS = {
```

Comment:
Can you please update this according to the changes suggested above?

Reply:
I've updated the views.py and configuration.rst accordingly, also the forms.rst but I'm unsure if I should split authentificate and reauthtificate in it because it is just one form.